app/interceptors/ExceptionInterceptor.java

```diff
@@ -72,7 +71,7 @@ private int getStatusCode(final Throwable throwable) {
     private String getMessage(final Throwable throwable, final int status) {
         String message = throwable.getMessage();
         if (StringUtils.isBlank(message)) {
-            message = HttpStatus.getStatusText(status);
+            message = Integer.toString(status);
```

Comment:
To be clear, if we use Apache httpcore's HttpStatus, it is transitive, we already have it.  Jetty is a new dependency.  I suspect the reason you see it in Eclipse is because it is used by some Eclipse plugins.